config/jobs/kubernetes/sig-network/ingress-nginx.yaml

```diff
@@ -0,0 +1,206 @@
+presubmits:
+
+  kubernetes/ingress-nginx:
+  - name: pull-ingress-nginx-boilerplate
+    always_run: true
+    decorate: true
+    path_alias: k8s.io/ingress-nginx
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+      - image: quay.io/kubernetes-ingress-controller/e2e:v05132019-4a7d93287
```

Comment:
Sorry adding a dumb question ---- should we host these images on `gcr.io/k8s-testimages` or somewhere similar?

Reply:
cc @krzyzacy

Reply:
> should we host these images on gcr.io/k8s-testimages or somewhere similar?

Not sure that makes sense (at least for now) because this image extends the ingress controller image.

Reply:
assume this image is public